src/main/java/edu/harvard/iq/dataverse/search/IndexBatchServiceBean.java

```diff
@@ -50,35 +50,35 @@ public class IndexBatchServiceBean {
     public Future<JsonObjectBuilder> indexStatus() {
         JsonObjectBuilder response = Json.createObjectBuilder();
         logger.info("Beginning indexStatus()");
-        JsonObject contentInDatabaseButStaleInOrMissingFromSolr = getContentInDatabaseButStaleInOrMissingFromSolr().build();
-        JsonObject contentInSolrButNotDatabase = null;
-        JsonObject permissionsInSolrButNotDatabase = null;
         try {
+            JsonObject contentInDatabaseButStaleInOrMissingFromSolr = getContentInDatabaseButStaleInOrMissingFromSolr().build();
+            JsonObject contentInSolrButNotDatabase = null;
+            JsonObject permissionsInSolrButNotDatabase = null;
+
             contentInSolrButNotDatabase = getContentInSolrButNotDatabase().build();
```

Comment:
Dumb question, but in 55,56 why not have the "build" in the same line as declaration as in 54?